hello.java

```diff
@@ -1,5 +1,5 @@
 public class Hello{

```

Comment:
A public class must be in a file with same name as the class. (Your filename is hello not Hello.) A class that is not public does not have to be same as file name.